test/integration/resize-handler.spec.ts

```diff
@@ -36,10 +37,10 @@ test('resize handler /content/:hash/dxt/:length', function ({ components, spyCom
   it('fails when auth chain is present but metadata.signer is invalid', async () => {
     const { localFetch } = components
 
-    const path = '/content/aHash/dxt/128'
+    const path = '/content/dxt/128'
     const request = await getSignedFetchRequest(path, { intent: VALID_CUSTOM_METADATA.intent, signer: 'invalid' })
 
-    const r = await localFetch.fetch(path, {
+    const r = await localFetch.fetch(path + '?asset=aHash', {
```

Comment:
Does this still accept a hash? Or is it always an asset url? If so, let's rename all `aHash` to `aUrl` to avoid misinterpretation.